networking/k8s_nmstate/k8s-nmstate-about-the-k8s-nmstate-operator.adoc

```diff
@@ -18,7 +18,7 @@ Before you can use NMState with {product-title}, you must install the Kubernetes
 
 [NOTE]
 ====
-The Kubernetes NMState Operator updates the network configuration of a secondary NIC. It cannot update the network configuration of the primary NIC or the `br-ex` bridge.
+The Kubernetes NMState Operator updates the network configuration of a secondary NIC. It cannot update the network configuration of the primary NIC or the `br-ex` bridge. On a bare-metal platform, the Kubernetes NMState Operator can update the `br-ex` bridge network configuration.
```

Comment:
Similar as above, only on a baremetal with nmstate-configuration-based br-ex, the kubernetes nmstate operator can update the 'br-ex' bridge network configuration.

Reply:
Thanks, @qiowang721 . How about:

> The Kubernetes NMState Operator updates the network configuration of a secondary NIC. The Operator cannot update the network configuration of the primary NIC, or update the `br-ex` bridge on most on-premise networks. On a bare-metal platform, using the Kubernetes NMState Operator to update the `br-ex` bridge network configuration is only supported if you set the br-ex` bridge as the interface in the NMState configuration in your `NodeNetworkConfigurationPolicy` manifest file for your cluster.